internal/pkg/containers/cri/cri_test.go

```diff
@@ -197,14 +197,14 @@ func (suite *CRISuite) TestPods() {
 	suite.Assert().Equal(pods[0].Name, pods[0].Containers[0].Display)
 	suite.Assert().Equal(pods[0].Name, pods[0].Containers[0].Name)
 	suite.Assert().Equal("SANDBOX_READY", pods[0].Containers[0].Status)
-	suite.Assert().Equal(pauseImageDigest, pods[0].Containers[0].Digest)
-	suite.Assert().Equal(pauseImage, pods[0].Containers[0].Image)
+	// suite.Assert().Equal(pauseImageDigest, pods[0].Containers[0].Digest)
```

Comment:
I dug into this and can talk about it at our next meeting. I don't think this should block moving to container 1.3.

Reply:
Want to log an issue to follow up on the investigation?

Reply:
Still looking into it and if I can't fix this week then for sure.

Reply:
Is it changing every time or just different?

Reply:
It is changing every time, and cycles through 2 or 3 different strings.

Reply:
I think we could certainly remove these checks, they're more for completeness

but I wonder if CRI inspector doesn't show the containers as it should now?